electricpy/phasor.py

```diff
@@ -445,25 +481,41 @@ def phasorplot(phasor, title="Phasor Diagram", legend=False, bg=None,
         # Plot with labels
         if legend != False:
             if mag > tolerance:
-                hand = _plt.arrow(0, 0, ang_r, mag, color=colors[i],
-                                label=legend[i], linewidth=linewidth)
+                hand = _plt.arrow(
```

Comment:
- [ ] as with other places, this seems a little unnatural. Perhaps we could use the keywords for the variables, or reduce it to a single row of arguments?

Reply:
could just use `x, y` and `dx, dy`: https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.arrow.html